Libraries/Core/InitializeCore.js

```diff
@@ -116,6 +116,24 @@ if (!global.__fbDisableExceptionsManager) {
   ErrorUtils.setGlobalHandler(handleError);
 }
 
+const formatVersion = version =>
+  `${version.major}.${version.minor}.${version.patch}` +
+  (version.prerelease !== null ? `-rc.${version.prerelease}` : '');
```

Comment:
Just concatenate the prerelease (instead of assuming -rc, see comments later in this review)